### PR TITLE
Warn users when creating applications named 'app'.

### PR DESCRIPTION
**Pull request by teobugslayer**
See https://github.com/NativeScript/nativescript-cli/issues/760

### CODE REVIEW

File: lib/commands/create-project.ts [hunk 1]
```diff
@@ -12,6 +13,10 @@ export class ProjectCommandParameter implements ICommandParameter {
 				this.$errors.fail("You must specify <App name> when creating a new project.");
 			}
 
+			if (value === "app") {
```
Comment (ligaz) [inline lib/commands/create-project.ts]: If the value is `App` we will not show the warning but we have to.

Comment (teobugslayer) [inline lib/commands/create-project.ts]: Great catch, thanks!

**Comment by teobugslayer** [comment]
ping @ligaz

File: lib/commands/create-project.ts [hunk 2]
```diff
@@ -12,6 +13,10 @@ export class ProjectCommandParameter implements ICommandParameter {
 				this.$errors.fail("You must specify <App name> when creating a new project.");
 			}
 
+			if (value.toUpperCase() === "APP") {
```
Comment (rosen-vladimirov) [inline lib/commands/create-project.ts]: Consider checking the OS as well, the message is useful only on MacOS

**Comment by rosen-vladimirov** [comment]
:+1: But I think the message must be shown only on MacOS

File: lib/commands/create-project.ts (hunk 2 above)
Comment (teobugslayer) [inline lib/commands/create-project.ts]: The project may be shared through version control with other people who has Mac OS. I think it is best to be shown always.

**Comment by Fatme** [comment]
:+1:

**Pull request merged**